.github/workflows/staging-build-rpm.yml

```diff
@@ -587,9 +587,79 @@ jobs:
         with:
           working-directory: kibana/plugins/anomaly-detection-kibana-plugin
           command: yarn cy:run-with-security --config baseurl=http://localhost:5601
+          
+  Test-SEC-KIBANA:
+    needs: [Provision-Runners]
+    runs-on: [self-hosted, Linux, X64, odfe-rpm-sec-kibana]
+    name: Test-SEC-KIBANA
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [14]
+    steps:
+      - uses: actions/checkout@v1
+      
+      - name: Retrieve plugin tags 
+        run: |
+          echo "p_tag_sec_kibana=$(release-tools/scripts/plugin_tag.sh opendistro-for-elasticsearch/security-kibana-plugin)" >> $GITHUB_ENV
+          echo "es_version=$(release-tools/scripts/version-info.py --es)" >> $GITHUB_ENV
+
+      - name: Checkout Kibana
+        uses: actions/checkout@v2
+        with:
+          repository: opendistro-for-elasticsearch/kibana-oss
+          ref: ${{env.es_version}}
+          token: ${{ secrets.ODFE_RELEASE_BOT_PUBLIC_PRIVATE_READ_WRITE_TOKEN }}
+          path: kibana  
+
+      - uses: actions/checkout@v2
+        with:
+          repository: opendistro-for-elasticsearch/security-kibana-plugin
+          ref: ${{env.p_tag_sec_kibana}}
+          path: kibana/plugins/opendistro_security
+      
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Get node and yarn versions
+        id: node_yarn_versions
+        run: |
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
```

Comment:
Can we change nvm version to v0.37.2 , the v0.34.0 is release in 2019.

Reply:
Not required as of now. Its fine as it is just used to install node